WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -38,7 +38,7 @@ private async Task OnNextAsync()
 		await AutoLoginAsync();
 
 		Navigate().Clear();
-		UiContext.Navigate().To(_wallet);
+		UiContext.Navigate().To(_wallet!);
```

Comment:
We agreed back then when nullable got enabled in the project, that `!` should never be used. Rather check for null in the beginning of the method.

Reply:
Fixed in latest commit.